### PR TITLE
add crictl container command

### PR DESCRIPTION
**Pull request by heartlock**
Signed-off-by: heartlock <21521209@zju.edu.cn>

ref: https://github.com/kubernetes-incubator/cri-tools/issues/35
@feiskyer @Random-Liu

### CODE REVIEW

File: cmd/crictl/container.go [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+	"time"
+
+	"github.com/urfave/cli"
+	"golang.org/x/net/context"
+	pb "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
+)
+
+var runtimeContainerCommand = cli.Command{
+	Name:    "container",
+	Aliases: []string{"ctr"},
```
Comment (feiskyer) [inline cmd/crictl/container.go]: Add a usage message for this

File: cmd/crictl/container.go [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+		if !context.IsSet("pod") {
+			return fmt.Errorf("Please specify the id of the pod sandbox to which the container belongs via the --pod option")
+		}
+
+		opts := createOptions{
+			configPath: context.String("config"),
+			podID:      context.String("pod"),
+		}
+
+		// Test RuntimeServiceClient.CreateContainer
```
Comment (feiskyer) [inline cmd/crictl/container.go]: Why comment with `Test`?

File: cmd/crictl/container.go [hunk 3]
```diff
@@ -0,0 +1,388 @@
...
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "pod",
+			Usage: "the id of the pod sandbox to which the container belongs",
+		},
+		cli.StringFlag{
+			Name:  "config",
+			Value: "config.json",
+			Usage: "the path of a container config file",
+		},
```
Comment (feiskyer) [inline cmd/crictl/container.go]: please add a flag of sandbox config

File: cmd/crictl/container.go [hunk 4]
```diff
@@ -0,0 +1,388 @@
...
+	config, err := loadContainerConfig(opts.configPath)
+	if err != nil {
+		return err
+	}
+
+	r, err := client.CreateContainer(context.Background(), &pb.CreateContainerRequest{
+		PodSandboxId: opts.podID,
+		Config:       config,
+		// TODO(runcom): this is missing PodSandboxConfig!!!
+		// we should/could find a way to retrieve it from the fs and set it here
```
Comment (feiskyer) [inline cmd/crictl/container.go]: we should pass the config in from CLI

File: cmd/crictl/container.go (hunk 3 above)
Comment (heartlock) [inline cmd/crictl/container.go]: Is sandbox config optional？

Comment (feiskyer) [inline cmd/crictl/container.go]: No, it is always passed to runtime

**Comment by feiskyer** [comment]
@heartlock Also PR needs another rebase.

**Comment by heartlock** [comment]
update it! @feiskyer

**Comment by feiskyer** [comment]
Cool. LGTM.

**Pull request merged**